lib/tokenizers.js

```diff
@@ -60,30 +61,42 @@ export async function get({id, explain = false} = {}) {
 
   const record = await collection.findOne(query, {projection});
   if(!record) {
-    const details = {
-      httpStatusCode: 404,
-      public: true
-    };
-    throw new BedrockError(
+    const error = new BedrockError(
       'Tokenizer not found.',
-      'NotFoundError', details);
+      'InvalidStateError', {httpStatusCode: 503, public: true});
```

Comment:
This should still be a `NotFound` error. There's a comment that says it was changed back, but the diff is saying it wasn't.

Reply:
Fixed.